src/booleans.js

```diff
@@ -1,59 +1,181 @@
 export const negate = (a) => {
-  // your code here
+  if(a === true) {
+	return false
+} 
+else {
+	return true
+}
+
 };
 
 export const both = (a, b) => {
-  // your code here
+
+if(a && b) {
+	return true
+}
+else {
+	return false
+}  
+
 };
 
+
 export const either = (a, b) => {
-  // your code here
+if(a || b) {
```

Comment:
If you ever find yourself doing an `if bla then return true, else return false` then you can probably cut it down to one line. `if bla then return false, else return true` then utilise the not operator `!`.